pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -695,6 +697,7 @@ func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.C
 	if err != nil {
 		return errors.Wrap(err, "k.Transform failed")
 	}
+	selector := "io.kompose.service"
```

Comment:
not to nitpick, but should this instead be a constant and at the top of the file? (outside the scope) ping @kadel